clang/test/SemaSYCL/sycl-device.cpp

```diff
@@ -1,6 +1,8 @@
 // RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
 // RUN: %clang_cc1 -verify -DNO_SYCL %s
 
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -DNOT_STRICT -Wno-error=sycl-strict -Wno-sycl-strict %s
```

Comment:
Please, add a check for the case when compiler emits warnings.

Reply:
Done.